### PR TITLE
AC Automated Test - LRAC-10869 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10869 (Automation ticket)
https://issues.liferay.com/browse/LRAC-10698 (Story)
[Testray](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1658039506)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by joshitagaki — APPROVED** [review]
Nice! LGTM

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase [hunk 1]
```diff
@@ -317,4 +317,43 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-10698 | Automation ID: LRAC-10869 | Test Summary: Empty state for all workspace pages"
+	@priority = "2"
+	test CheckAllMessageWhenNoPropertyNoDataSource {
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:322]: Hey Vini, maybe replace `CheckAllMessageWhenNoPropertyNoDataSource` with `CheckAllMessagesWhenNoPropertyNoDataSource` would be better?

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase:322]: Hey Manu, I think the best name is `CheckMessageWhenNoPropertyNoDataSource`, What do you think?

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/EmptyState.testcase]: Yes, even better.

**Review by manuferreira — APPROVED** [review]
LGTM, thanks Vini.

**Comment by ViniLopes20** [comment]
Hi team, Thanks for reviewing!

I'm forwarding it!

**Comment by ViniLopes20** [comment]
ci:forward

**Pull request closed without merging**